wait/host_port.go

```diff
@@ -209,7 +209,7 @@ func internalCheck(ctx context.Context, internalPort nat.Port, target StrategyTa
 
 		if exitCode == 0 {
 			break
-		} else if exitCode == 126 {
+		} else if exitCode == 126 || exitCode == 127 {
```

Comment:
suggestion: Could we add a comment about why we have multiple values and extract them to constants so they are not magic numbers any more.
126 is also used in `host_port_test.go` too.

You can find the details of what they [represent here](https://github.com/moby/moby/blob/b3c775059314039807862defe9f531c2bca049db/daemon/errors.go#L122).

Reply:
Done.